tests/commit.bats

```diff
@@ -91,3 +91,8 @@ load helpers
   [ "${status}" -eq 0 ]
   [ "$output" == "/bin/sh" ]
 }
+
+@test "commit-no-name" {
+  cid=$(buildah from --pull --signature-policy ${TESTSDIR}/policy.json alpine)
+  run_buildah commit --signature-policy ${TESTSDIR}/policy.json $cid
+}
```

Comment:
can you check the status at least here and maybe output?

Reply:
run_output takes care of this for you.  I believe it makes sure that you exit with 0 and always echos $output